src/main/java/io/confluent/idesidecar/restapi/exceptions/ParsingFeatureFlagsFailedException.java

```diff
@@ -0,0 +1,12 @@
+package io.confluent.idesidecar.restapi.exceptions;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.quarkus.runtime.annotations.RegisterForReflection;
+
+@RegisterForReflection
+@JsonInclude(JsonInclude.Include.ALWAYS)
+public record ParsingFeatureFlagsFailedException(
+    String code,
+    String message
+){
+}
```

Comment:
Good point. I'm going to refactor this, by moving the record into the `HttpFeatureFlagProvider` and renaming it (since it represents the error response from LD and shouldn't exist outside of that provider, and because that's where the `FeatureFlagFailureException` are instantiated). I'll also change the `FeatureFlagFailureException` to have a `code` field rather than the record type.

Reply:
Made this change.